ecf/day.sh

```diff
@@ -11,14 +12,34 @@ set -e
 echo zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz loading modules zzzzzzzzzzzzzzzzzzzzzzz
 #. /usrx/local/Modules/3.2.10/init/bash
 module purge
-module load ips/19.0.5.281 impi/19.0.5
-module load prod_envir/1.1.0
-module load prod_util/1.1.5 
-module load grib_util/1.1.1
-module load bufr_dumplist/2.3.0
-module load dumpjb/5.1.0
-module load NetCDF/4.5.0
-module load lsf/10.1
+module load envvar/1.0
+module load imagemagick/7.0.8-7
+module load prod_envir/2.0.5
+module load prod_util/2.0.9
+#Can be loaded directly -- required for additional modules
+module load intel/19.1.3.304
+module load craype/2.7.8
+module load PrgEnv-intel/8.1.0
+#Additional:
+module load wgrib2/2.0.8
+module load w3nco/2.4.1
+
+module load jasper/2.0.25
+module load libpng/1.6.37
+module load zlib/1.2.11
+module load libjpeg/9c
+module load bacio/2.4.1
+module load bufr/11.5.0
```

Comment:
perhaps try newer versions 11.7.0+.  A new 12.0.0 is in the making (FYI)

Reply:
Already at 11.7.0 in my copy. Before implementation, I always go to the most recent possible.